<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>108</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Download</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="pointlLayout">
     <item>
      <widget class="QLabel" name="pointLabel">
       <property name="text">
        <string>Point</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pointCountLabel">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="pointBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="tracklLayout">
     <item>
      <widget class="QLabel" name="trackLabel">
       <property name="text">
        <string>Track</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="trackCountLabel">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="trackBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>pointBar</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>trackBar</zorder>
  <zorder>horizontalLayoutWidget_2</zorder>
  <zorder>trackCountLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
